### ISSUE TITLE
RPC logging level inconsistency

### ISSUE THREAD

**Issue by abitmore** [bitshares/bitshares-core#929]
As described in https://github.com/bitshares/bitshares-core/issues/802#issuecomment-378030241, RPC related logging level is:
* `level=error` : logs errors
* `level=warn` : logs requests
* `level=info` : logs responses

However, it only applies to websocket connection, as mentioned in https://github.com/bitshares/bitshares-core/pull/919#pullrequestreview-120697056:
> For simple requests with curl I didn't see responses being logged at level "info".

## CORE TEAM TASK LIST
- [x] Evaluate / Prioritize Feature Request
- [x] Refine User Stories / Requirements
  - [x] Assignee: @cogutvalera 
  - [x] Estimation: 2 hours
- [ ] Define Test Cases
- [ ] Design / Develop Solution
- [ ] Perform QA/Testing
- [ ] Update Documentation

**Comment by nanomobile** [comment]
@ryanRfox can I claim this issue please ?

**Comment by ryanRfox** [comment]
Yes, I'll move it to Community Claims for you @nanomobile . Prior to beginning work, we need to establish an estimate on the effort. May I request you to provide your estimate here? I will work with the Core Team to confirm, then you may begin development based on that estimate. Thanks

**Comment by nanomobile** [comment]
Thank you very much ! A bit later I will provide my estimate here ...

**Comment by nanomobile** [comment]
my estimation for this issue is approximately 1/2 hour. @ryanRfox @jmjatlanta @oxarbitrage @clockworkgr @abitmore @xeroc what is your approximation ?

**Comment by abitmore** [comment]
I'm fine with that.

**Comment by nanomobile** [comment]
I don't know if 2 hours will be enough or not for this issue, it's just my approximation :-) maybe your approximation guys would be much better suited for this issue

**Comment by cogutvalera** [comment]
I need to resolve #1141 issue and only after that I will can resolve this issue

**Comment by cogutvalera** [comment]
breakpoints work on wscat requests but not on cURL requests. For example exactly this line of code https://github.com/bitshares/bitshares-fc/blob/0a90eff69778c0ef1d26f75d11d459ca172eff0c/src/network/http/websocket.cpp#L161 can be stopped by breakpoint for wscat requests on sending result, but on cURL requests breakpoints here won't work.

Can anybody help me to find exactly the line of code where response is sending on cURL requests for wintess_node ?

**Comment by abitmore** [comment]
Forwarded comment/question from #1141:
>Also @abitmore I don't want to debug with a specific tool, I just want to debug with any tool but I've tried different debugging tools and cannot debug just some little part in fc library, at the same time other parts of the same fc library I can debug with different debugging tools. @abitmore can you help me please to debug with any debugging tool specific cURL requests for witness_node ? which tool do you suggest for this purpose ? I think if it is really basic programming skills for you than you should help me and after that close this issue :-)

Sorry, to me, this issue is low priority, so I won't spend time on it right now. You claimed it, that means it's your issue, best if you can get it resolved with your own efforts, nobody else is responsible for helping you, although someone will help sometimes.

**Comment by abitmore** [comment]
Another comment forwarded from #1141:
> Common causes of breakpoints not working:
>- The line was never run. This is a common problem that catches even the best developers at times. Step into the function call instead of using breakpoints, and make sure the method you expect to have been called is actually called.
>- library not built with debugging information (probably not the case here, as you can debug other stuff in the same library)
>- code not compiled in ( line 17 is a blank line according to the debugging information, but is an actual line in your new code that has yet to be compiled with debugging information turned on)

@nanomobile please read it, especially the first cause, I guess you've encountered it.

**Comment by cogutvalera** [comment]
@abitmore I don't want that you help me right now, I'm always trying with my own efforts, just want to make informational discussion history here for all other developers maybe will be useful later ...
While nobody can help me of course I will try to solve all my issues only by my own efforts as soon as possible ! Hope I will solve it by myself much earlier than help will come :-) I'm writing here in github without meaning that somebody must answer right now to me or right now must help me, no no no :-) just want to make some kind of documentation, while trying to solve my issues with my own efforts only, and maybe somebody knows the answer because had similar experience earlier :-) and can help me just by writing 1 simple sentence only :-) I don't want to waste your time or any others time !

**Comment by cogutvalera** [comment]
I should shape my words another way without "can anybody help me" so people will understand my messages or questions more correctly

**Comment by cogutvalera** [comment]
https://github.com/bitshares/bitshares-fc/pull/62

**Comment by ryanRfox** [comment]
Added assignment to @cogutvalera and estimation of 2 horus to Description.

**Comment by ryanRfox** [comment]
fixed by` RPC logging level inconsistency #62` /bitshares/bitshares-fc/#62

**Comment by cogutvalera** [comment]
@abitmore  still I need to bump FC here ? Thanks !

**Comment by cogutvalera** [comment]
Thanks !

**Comment by abitmore** [comment]
Fixed with #1104.
